pages/proof-of-yield/[timestamp].tsx

```diff
@@ -0,0 +1,92 @@
+import Head from "next/head";
+import React from "react";
+import Error from "../404";
+import moment from "moment";
+import { get } from "lodash";
+import { useRouter } from "next/router";
+import {
+  DayBasicData,
+  DayDripperBanner,
+  DayOtherSources,
+  DayStrategyPerformance,
+  DayTotal,
+} from "../../sections";
+import { DailyStat, YieldOnDayProps } from "../../types";
+import { GetServerSideProps, GetServerSidePropsContext } from "next";
+import { fetchAPI, fetchDailyStats, transformLinks } from "../../utils";
+import { Header, Footer } from "../../components";
+
+const overrideCss = "px-8 md:px-10 lg:px-10 xl:px-[8.375rem]";
+
+const YieldOnDay = ({ navLinks, dailyStat }: YieldOnDayProps) => {
+  const router = useRouter();
+  let { timestamp } = router.query;
+
+  const timestampNumber = Number(timestamp as any);
+
+  if (Number.isNaN(timestampNumber)) return <Error navLinks={navLinks} />;
+
+  return (
+    <>
+      <Head>
+        <title>Proof of Yield</title>
+      </Head>
+
+      <Header mappedLinks={navLinks} background="bg-origin-bg-black" />
+
+      <DayBasicData
+        timestamp={timestampNumber}
+        dailyStat={dailyStat}
+        sectionOverrideCss={overrideCss}
+      />
+
+      {/* <DayDripperBanner sectionOverrideCss={overrideCss} />
+
+      <DayStrategyPerformance sectionOverrideCss={overrideCss} />
+
+      <DayOtherSources sectionOverrideCss={overrideCss} />
+
+      <DayTotal sectionOverrideCss={overrideCss} /> */}
+
+      <Footer />
+    </>
+  );
+};
+
+export const getServerSideProps: GetServerSideProps = async (
+  context: GetServerSidePropsContext<{ timestamp: string }>
+): Promise<{
+  props: YieldOnDayProps;
+}> => {
+  const { timestamp } = context.query;
+
+  let timestampNumber: number | undefined;
+  let dailyStat: DailyStat;
+  if (
+    timestamp &&
+    typeof timestamp === "string" &&
+    (timestampNumber = parseInt(timestamp))
```

Comment:
This feels like a bit of an odd syntax to put into the conditional. Is there a particular reason to set the value within it? 

If not, ideally we move this out into the next line within the brackets.